sources/bash_completion.d/swizzin

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# ln -fs /etc/swizzin/sources/bash_completion.d/swizzin /etc/bash_completion.d/swizzin
+#
+_command_complete() {
+    local cur prev
+    cur=${COMP_WORDS[COMP_CWORD]}
+    prev=${COMP_WORDS[COMP_CWORD - 1]}
+    readarray -d ' ' apps_array < <(find "/etc/swizzin/scripts/install/" -maxdepth 1 -mindepth 1 -type f -printf "%f ")
+    readarray -d ' ' upgrade_array < <(find "/etc/swizzin/scripts/upgrade/" -maxdepth 1 -mindepth 1 -type f -printf "%f ")
+
+    case ${COMP_CWORD} in
+        1)
+            mapfile -t COMPREPLY < <(compgen -W "install remove adduser deluser chpasswd update upgrade panel list" -- "${cur}")
```

Comment:
Might be nice to organise this by "topic" a little, and added some others left.
```suggestion
            mapfile -t COMPREPLY < <(compgen -W "install remove upgrade update adduser deluser chpasswd  panel rtx rmgrsec list help" -- "${cur}")
```

Reply:
compgen will sort it automatically.

Reply:
Yeah, the order is lost, shoud still add the rest of the missing commands. I changed the order back to just show the additions